go.mod

```diff
@@ -25,3 +25,5 @@ require (
 	gonum.org/v1/gonum v0.11.0
 	google.golang.org/grpc v1.45.0
 )
+
+replace github.com/onflow/flow-emulator => github.com/findonflow/flow-emulator custom-zerolog
```

Comment:
Let me know once emulator PR is in and this is removed so I approve the PR.

Reply:
emulator PR is now in but emulator has not been release yet AFAIK @sideninja

Reply:
Hey, @bjartek - @sideninja is away for a few days but coming back on the 14th, so he should be able to prioritize this once he's back.